packages/nc-gui/plugins/projectLoader.js

```diff
@@ -63,22 +63,23 @@ export default async({ store, redirect, $axios, $toast, $api, route }) => {
   // fetch latest release info
   const fetchReleaseInfo = async() => {
     try {
-      const releaseInfo = await $api.utils.appInfo()
-      const latestRelease = await $api.utils.appVersion()
-      if (releaseInfo && latestRelease && releaseInfo.version) {
-        store.commit('app/MutReleaseVersion', releaseInfo.version)
-        store.commit('app/MutLatestRelease', latestRelease.releaseVersion || null)
+      const versionInfo = await $api.utils.appVersion()
+      if (versionInfo &&
+          versionInfo.releaseVersion &&
+          versionInfo.currentVersion &&
+          !(/[^0-9.]/.test(versionInfo.currentVersion))) {
+        store.commit('app/MutCurrentVersion', versionInfo.currentVersion)
+        store.commit('app/MutLatestRelease', versionInfo.releaseVersion || null)
```

Comment:
I think `versionInfo.releaseVersion` is already enough instead of `versionInfo.releaseVersion || null`.